risc0/zkvm/sdk/rust/host/src/lib.rs

```diff
@@ -25,6 +25,9 @@ pub use exception::Exception;
 #[cxx::bridge]
 mod bridge {}
 
+/// The default digest count when generating a MethodId.
```

Comment:
This constant seems to pop up a lot; is there a way to share it between locations?

Reply:
This was just a turd, thanks for catching it.

Reply:
I don't know how to share constants between C++ and Rust (unfortunately).

Once we finish the Rust port of the prover, a lot of this can be cleaned up.